config/application.rb

```diff
@@ -8,10 +8,12 @@
 
 module Theodinproject
   class Application < Rails::Application
-    require Rails.root.join("lib/custom_public_exceptions")
+    require Rails.root.join('lib/custom_public_exceptions')
     config.exceptions_app = CustomPublicExceptions.new(Rails.public_path)
     config.middleware.insert_after ActionDispatch::Static, Rack::Deflater
 
+    config.action_controller.raise_on_unfiltered_parameters = true
```

Comment:
Have you tested the login, signup and settings form still works after making this change?

Reply:
Not sure how this change got checked in. Will remove it.